cli/pkg/kctrl/cmd/package/repository/list.go

```diff
@@ -113,7 +113,13 @@ func NewSourceValue(pkgr v1alpha1.PackageRepository) uitable.Value {
 	if pkgr.Spec.Fetch != nil {
 		switch {
 		case pkgr.Spec.Fetch.ImgpkgBundle != nil:
-			source = "(imgpkg) " + pkgr.Spec.Fetch.ImgpkgBundle.Image
+			tag, ok := pkgr.Annotations["tag"]
```

Comment:
Should use a constant defined in the part of the code which is adding the annotation

Reply:
Adopted